src/Nncase.Quantization/Quantization/Quantizer.cs

```diff
@@ -154,6 +169,82 @@ private ValueRange<float> FixUpRange(ValueRange<float> range, bool symmetric = f
         return ranges;
     }
 
+    private IDictionary<ENode, ValueRange<float>[]> GetRangesFromConfig(string quantScheme)
+    {
+        var ranges = new Dictionary<ENode, ValueRange<float>[]>(ReferenceEqualityComparer.Instance);
+        string readJson = quantScheme;
+        dynamic configJson = Newtonsoft.Json.Linq.JObject.Parse(readJson);
```

Comment:
Performance of `dynamic` is slow. It's better to write the scheme record classes and using `JsonConvert.DeserializeObject<QuantScheme>`.